test/test_modulations.jl

```diff
@@ -0,0 +1,26 @@
+include("../src/enums.jl")
+include("../src/event.jl")
+@testset "InfectionModulations" begin
+  state = MocosSim.SimState(3)
+  params = Missing
```

Comment:
`Missing` is a type, the value is `missing` and it is of type `Missing` hence would not fit to the signature of the function